.gitignore

```diff
@@ -37,3 +37,9 @@ next-env.d.ts
 
 # Local Netlify folder
 .netlify
+
```

Comment:
I'm not sure if it's necessary to ignore additional `.env` files.  We want to restrict AppDevs from committing their .env file, which is the completed version of the .env.development template.  We're not ignoring the `.env.development` file because it serves as the blueprint for running the app locally, with placeholder values.  The README was updated earlier to reflect this.  If you have any questions, feel free to reach out, and I can explain it in more detail on a call.